tests/classes/static_properties_003.phpt

```diff
@@ -2,7 +2,7 @@
 Attempting to access static properties using instance property syntax
 --FILE--
 <?php
-class C {
+#[AllowDynamicProperties] class C {
```

Comment:
nit: this would be a syntax error in php 7.x and older and my preference is to avoid this as an example for test cases for other applications/libraries/PECLs (until php 7.x stops receiving security support and most code highlighters are updated to be aware of attribute syntax)

Reply:
I've put the attribute on the same line to avoid updating line numbers in the output...